packages/mobile/src/backup/__snapshots__/Backup.test.tsx.snap

```diff
@@ -253,7 +253,7 @@ exports[`Backup renders correctly if backup not completed 1`] = `
   <View
     style={
       Object {
-        "height": 45,
+        "height": 56,
```

Comment:
Wasn't expecting changes with this, can you confirm the backup flow looks all good?

Reply:
Going through it it looked on a Nexus 5.

![Screenshot_1567784618](https://user-images.githubusercontent.com/57791/64441289-df9d1e80-d0cd-11e9-8654-a65d0ed94ac4.png)
![Screenshot_1567784469](https://user-images.githubusercontent.com/57791/64441258-ceeca880-d0cd-11e9-8e07-74f609412cae.png)
![Screenshot_1567784495](https://user-images.githubusercontent.com/57791/64441214-b9777e80-d0cd-11e9-89a1-967e8f937828.png)